src/installation/guides/fde.md

```diff
@@ -274,6 +274,12 @@ Next, install the boot loader to the disk.
 [xchroot /mnt] # grub-install /dev/sda
 ```
 
+Or, for UEFI systems.
+
+```
+[xchroot /mnt] # grub-install --target=x86_64-efi --efi-directory=/efi
```

Comment:
Why are you forcing ESP to `/efi`?

Reply:
Doesn't work otherwise.
However I learned it from using gentoo, can't tell if grub would work without it.

Reply:
this should probably match the [regular chroot install guide](https://docs.voidlinux.org/installation/guides/chroot.html#installing-grub)

Reply:
Will do

Reply:
Deleted --efi-directory=/efi to match the chroot install guide.